hashicorp/web-unified-docs
# Chore: remove Nomad content

# Changes

- Remove Nomad content from code base to prep for education engineer test. This includes:
  - Removing nomad from the `content` dir
  - Removing nomad from the `public/assets` dir

- [Asana task](https://app.asana.com/0/1207899860738460/1208561647881340/f)

# Testing

1. Make a curl request or use Postman to send a `GET` request 
```
curl https://web-presence-experimental-docs.vercel.app/api/content/nomad/nav-data/latest/docs
```
1. You should receive a valid response, such as this:
<img width="1143" alt="nomad" src="https://github.com/user-attachments/assets/71c45675-6191-4a49-9fa1-359e4e77ec87">


1. Make a curl request or use Postman to send a `GET` request
```
curl https://web-presence-experimental-docs-git-heat-chorer-9485c3-hashicorp.vercel.app/api/content/nomad/nav-data/latest/docs
```
1. You should receive a response of `Not Found`
